spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkWriteConf.java

```diff
@@ -236,4 +237,19 @@ public boolean useTableDistributionAndOrdering() {
         .defaultValue(SparkWriteOptions.USE_TABLE_DISTRIBUTION_AND_ORDERING_DEFAULT)
         .parse();
   }
+
+  public long validateFromSnapshotId() {
+    return confParser.longConf()
+        .option(SparkWriteOptions.VALIDATE_FROM_SNAPSHOT_ID)
+        .defaultValue(0)
```

Comment:
I don't think 0 is a correct default. If we set it to 0, the validation will throw an exception that it cannot find a snapshot with ID = 0. Should this use `Long` and `parseOptional`? The place that calls it will have to be adapted too.